plextraktsync/sync.py

```diff
@@ -126,28 +126,37 @@ def trakt_wl_shows(self) -> Dict[int, TVShow]:
 
     def sync(self, walker: Walker, dry_run=False):
         listutil = TraktListUtil()
+        is_partial = walker.is_partial
 
         if self.config.update_plex_wl_as_pl:
-            listutil.addList(None, "Trakt Watchlist", trakt_list=self.trakt.watchlist_movies)
+            if is_partial:
+                logger.warning("Running partial library sync. Watchlist as playlist won't update because it needs full library sync.")
+            else:
+                listutil.addList(None, "Trakt Watchlist", trakt_list=self.trakt.watchlist_movies)
 
         if self.config.sync_liked_lists:
-            for lst in self.trakt.liked_lists:
-                listutil.addList(lst["listid"], lst["listname"])
+            if is_partial:
+                logger.warning("Partial walk, disabling liked lists updating")
```

Comment:
```suggestion
                logger.warning("Partial walk, disabling liked lists updating. Liked lists won't update because it needs full library sync.")
```